Make pagination pretty

mchuong1 opened the pull request.

mchuong1 committed:
Add key to div

mchuong1 committed:
add in white background with box-shadow

mchuong1 committed:
Merge branch 'master' into make-pagination-pretty

mchuong1 committed:
add width: fit-content

remove commented css

mchuong1 committed:
Add list animation

mchuong1 committed:
put ++ operator inside props

mchuong1 committed:
Update plane icon in JobOverview

t-schell commented on src/css/SearchResults/SearchResults.css:
```diff
@@ -13,11 +13,29 @@
-    height: 284px;
+    height: 284px; 
+    animation-name: animateIn;
+    animation-duration: 350ms;
```
I like the animation. IMO this would look a little smoother if it were faster. When using the bottom pagination there's about a second or two before you start seeing the cards become visible.

t-schell commented on src/css/SearchResults/SearchResults.css:
```diff
@@ -150,18 +168,25 @@
 #card-intro a:hover{
     cursor: pointer;
+}
+.pagination-row{
```
Again, this is just IMO but I think it could look cleaner if we had the pagination component centered. I know this would awkwardly put the results count text over on the left but I don't think it would look bad if we swapped the two. So basically pagination would be centered just below the navbar and then the page results text would be left-aligned right above the first card. Let me know of your thoughts here.

![image](https://user-images.githubusercontent.com/41658758/110419622-b74a0380-8067-11eb-928d-beed098045a6.png)

mchuong1 committed:
Swap pagination and results count

add some spacing at the bottom of the page
adjust spacing between pagination at top as well

mchuong1 committed:
Make animation faster

t-schell reviewed (approved):
New changes look good. Pulling this in.
